common/reviews/api/heft.api.md

```diff
@@ -54,7 +54,10 @@ export { CommandLineStringListParameter }
 export { CommandLineStringParameter }
 
 // @public
-export type GlobChangedFilesFn = (patterns: string | string[], options?: IGlobChangedFilesOptions) => string[];
+export type GlobFn = (pattern: string | string[], options?: IPartialGlobOptions | undefined) => Promise<string[]>;
+
+// @public
+export type GlobSyncFn = (pattern: string | string[], options?: IPartialGlobOptions | undefined) => string[];
```

Comment:
```suggestion
export type GlobFn = (pattern: string | string[], options?: IPartialGlobOptions | undefined) => Promise<string[]> | string[];
```

Do we actually care if it's async or not? Aren't we going just await this?

Reply:
I made both types because I use the `getFilePathsAsync` method in FileGlobSpecifier.ts both for async and sync contexts. Sometimes the actual glob method is passed in (ex. in `CopyFilesPlugin.copyFilesAsync`), and sometimes it uses the sync method (ex. in `CopyFilesPlugin.copyIncrementalFilesAsync`, which eventually does async file copies)

Reply:
Though I see what you're saying... it can just be the async type and not do any async work. I'll remove.